runtime/worker.rs

```diff
@@ -83,6 +84,32 @@ pub struct WorkerOptions {
   pub stdio: Stdio,
 }
 
+impl Default for WorkerOptions {
+    fn default() -> Self {
+        Self {
+            web_worker_preload_module_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
+            create_web_worker_cb: Arc::new(|_| unimplemented!("web workers are not supported")),
+            module_loader: Rc::new(FsModuleLoader),
+            seed: None,
```

Comment:
I've explicitly specified seed as `None` since it could potentially be a really bad idea if its type is changed and it just falls back to a numeric default like `0`.